include/fmt/format-inl.h

```diff
@@ -2358,13 +2371,17 @@ int format_float(T value, int precision, float_specs specs, buffer<char>& buf) {
       return 0;
     }
     buf.try_resize(to_unsigned(precision));
-    std::uninitialized_fill_n(buf.data(), precision, '0');
+    if (is_constant_evaluated()) {
+      fill_n(buf.data(), precision, '0');
+    } else {
+      std::uninitialized_fill_n(buf.data(), precision, '0');
+    }
```

Comment:
I think you can call `fill_n` unconditionally here.

Reply:
Now I call it unconditionally :+1: